base/common/src/test/java/com/netscape/certsrv/cert/CertDataTest.java

```diff
@@ -0,0 +1,68 @@
+package com.netscape.certsrv.cert;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mozilla.jss.netscape.security.util.Cert;
+
+import com.netscape.certsrv.dbs.certdb.CertId;
+
+public class CertDataTest {
+
+    private static CertData before = new CertData();
+    private static StringWriter sw = new StringWriter();
+    private static PrintWriter out = new PrintWriter(sw, true);
```

Comment:
IIUC the `@Before` method will be executed before each test, so would it add duplicate cert into the `StringWriter` object?

Reply:
Yes, sorry, too much copy pasta - I should have used `BeforeClass`. We should probably start using `JUnit5` anyway; then it would be the clearer `@BeforeEach/@BeforeAll` instead of the more cryptic `@Before/@BeforeClass`.